source/sites/mon-entreprise.fr/pages/Stats/Stats.tsx

```diff
@@ -1,14 +1,76 @@
 import { ScrollToTop } from 'Components/utils/Scroll'
 import React from 'react'
 import emoji from 'react-easy-emoji'
+import {
+	Bar,
+	BarChart,
+	CartesianGrid,
+	Line,
+	LineChart,
+	Tooltip,
+	XAxis,
+	YAxis
+} from 'recharts'
+import * as data from '../../../../data/stats.json'
+import * as uniq_visitors from '../../../../data/total_visits.json'
 
 export default function Stats() {
+	let d202004 = data[11].values
+	console.log('data202004', d202004)
 	return (
 		<>
 			<ScrollToTop />
 			<h1>
 				Statistiques <>{emoji('📊')}</>
 			</h1>
+			<section>
+				<h2>Nombre de visites par mois</h2>
+				<LineChart
+					width={700}
+					height={300}
+					data={uniq_visitors.slice()}
```

Comment:
Pourquoi `.slice()` ?

Reply:
J'avais une erreur _Invalid prop `data` of type `object` supplied to `LineChart`, expected an array._ si je mets pas .slice()

Reply:
Si tu vas sur la version déployée et que tu regardes dans la console, on voit une erreur liée à ça https://deploy-preview-969--syso.netlify.com/stats

Reply:
Ok pour le slice il faut remplacer l'import 

```js
import * as uniq_visitors from '../../../../data/total_visits.json'
```

par

```js
import uniq_visitors from '../../../../data/total_visits.json'
```